dwave/cloud/cli.py

```diff
@@ -419,21 +429,32 @@ def standardized_output(fn):
     def wrapped(*args, **kwargs):
         # text/json output taken from callee args
         json_output = kwargs.get('json_output', False)
+        raw_output = kwargs.get('raw_output', False)
 
         now = utcnow()
         info = dict(datetime=now.isoformat(), timestamp=datetime_to_timestamp(now), code=0)
+        raw_stream = io.StringIO()
 
-        def output(fmt, maxlen=None, **params):
+        def output(fmt=None, maxlen=None, **params):
+            if raw := params.pop('raw', None):
+                raw_stream.write(str(raw))
             info.update(params)
-            if not json_output:
-                msg = fmt.format(**params)
+            if not json_output and not raw_output:
+                if fmt is not None:
+                    msg = fmt.format(**params)
+                else:
+                    # logfmt-like output
+                    msg = ' '.join(f'{k}={v}' for k,v in params.items())
                 if maxlen is not None:
                     msg = strtrunc(msg, maxlen)
                 click.echo(msg)
 
         def flush():
-            if json_output:
+            if json_output and not raw_output:
                 click.echo(orjson.dumps(info))
+            if raw_output and raw_stream.tell():
+                raw_stream.seek(0)
+                click.echo(raw_stream.read())
```

Comment:
Why use `StringIO` instead of just `click.echo(info.get("raw"))` here?

Reply:
You could then also just have an elif instead
```python
if raw_output:
    ...
elif json_output:
    ...
```

Reply:
The reason for using a string buffer here was to enable accumulation with `output()` calls and dumping all at the end in `flush()`, i.e. the same logic we use for JSON output. But we don't need buffering for the raw output at the moment, so sure, we can simplify the code a bit. Might go back to this implementation when/if we need buffered raw output from the CLI in the futuer.